### PR TITLE
feat: Added functionality

### PR DESCRIPTION
**Pull request by applike-ss**
## How Has This Been Tested?
- [ ] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [ ] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [ ] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: data.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+module "ssm_label" {
+  source  = "cloudposse/label/null"
+  version = "0.25.0"
+
+  delimiter = "/"
+
+  context = module.this.context
+}
```
Comment (mfroembgen) [inline data.tf:8]: Does it make sense to have a `slash` label? Because it is the same as the autoscaling one 🤔

File: examples/basic/versions.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "4.45.0"
+    }
+  }
+
+  required_version = "1.3.6"
```
Comment (mfroembgen) [inline examples/basic/versions.tf:9]:
```suggestion
  required_version = ">= 1.3.6"
```

File: data.tf (hunk 1 above)
Comment (applike-ss) [reply data.tf:8]: I don't think it makes sense, because both serve a different purpose.

File: examples/basic/versions.tf (hunk 2 above)
Comment (applike-ss) [reply examples/basic/versions.tf]: i've pushed a fix for this one

**Review by mfroembgen — APPROVED** [review]

**Comment by mfroembgen** [comment]
This PR is included in version 1.0.0 :tada:

**Pull request merged**